packages/nextjs-template/components/DendronSearch.tsx

```diff
@@ -85,8 +85,9 @@ function DendronSearchComponent(props: DendronCommonProps & SearchProps) {
   );
   const [results, setResults] = React.useState<SearchMode>();
   const dispatch = useCombinedDispatch();
+  //const notes = useCombinedSelector((state) => state.engine.notes);
```

Comment:
remove commented code. There's also an unused import in this file for this function.